README.md

```diff
@@ -15,37 +41,68 @@ The DataRobot CLI (`dr`) is a command-line interface for managing DataRobot cust
 - 🔄 **Self-update capability**&mdash;easily update to the latest version with a single command.
 - 🎨 **Beautiful TUI**&mdash;terminal UI built with Bubble Tea for an enhanced user experience.
 
-## Table of Contents
+## Table of contents
 
 - [Installation](#installation)
-- [Quick Start](#quick-start)
-- [Documentation](#documentation)
-- [Commands](#commands)
-- [Shell Completion](#shell-completion)
+- [Quick start](#quick-start)
+- [Next steps](#next-steps)
 - [Contributing](#contributing)
 - [License](#license)
+<!--
+- [Commands](#commands)
+- [Shell completion](#shell-completion)
+- [Development](#development)
+- [Release](#release)
+-->
 
 ## Installation
 
-### Quick Install (Recommended)
-
 Install the latest version with a single command:
 
-#### macOS/Linux
+### macOS/Linux
 
 ```bash
 curl https://cli.datarobot.com/install | sh
 ```
 
-#### Windows (PowerShell)
+### Windows (PowerShell)
 
 ```powershell
 irm https://cli.datarobot.com/winstall | iex
 ```
 
-### Install Specific Version
+<details><summary><em>Alternative installation methods</em></summary>
```

Comment:
BTW (CC @ajalon1) do we want to include instructions for how to install using homebrew?